src/ovh-angular-pagination-front.html

```diff
@@ -1,36 +1,41 @@
 <div class="pagination-container">
 
-    <ul uib-pagination class="pagination-sm"
-                data-total-items="totalItems"
-                data-max-size="maxSize"
-                data-boundary-links="true"
-                data-rotate="false"
-                data-num-pages="numPages"
-                data-items-per-page="itemsPerPage"
-                data-ng-model="currentPage"
-                data-previous-text="&lsaquo;"
-                data-next-text="&rsaquo;"
-                data-first-text="&laquo;"
-                data-last-text="&raquo;"></ul>
+    <ul data-uib-pagination
+        data-total-items="totalItems"
+        data-max-size="maxSize"
+        data-boundary-links="true"
+        data-rotate="false"
+        data-num-pages="numPages"
+        data-items-per-page="itemsPerPage"
+        data-ng-model="currentPage"
+        data-previous-text="&lsaquo;"
+        data-next-text="&rsaquo;"
+        data-first-text="&laquo;"
+        data-last-text="&raquo;"></ul>
 
-    <input class="pagination-page-input pull-right"
-           data-ng-show="goToPage !== 'false'"
-           name="currentPageInput"
-           type="number"
-           min="1"
-           data-ng-model="goPage"
-           data-ng-model-options="{ debounce: {'default': 500} }"
-           data-ng-change="changePage()"
-           data-ng-blur="goPage = null"
-           data-ng-keypress="changePageKeyPress($event);"
-           placeholder="{{pagePlaceholder}}"/>
+    <form class="form-inline pull-right">
```

Comment:
This means that we cannot use this into a ```<form>``` tag. Are we sure that we already don't use this into this tag ?